### PR TITLE
add disruptive test and console test for logging 5.7

### PR DESCRIPTION
**Pull request by QiaolingTang**
1. Add new jobs to only test disruptive test cases for logging 5.7
2. fix failure in the scripts of subscribe operators when the cluster is a private cluster
3. use `cucushift-e2e-destructive` to run logging cases in cucushift as most of the test cases should be executed one-by-one
4. add web tests

### CODE REVIEW

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-ovn-ipsec-p1-f7-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-gcp-ipi-ovn-ipsec-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-gcp-ipi-proxy-private-p2-f14-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-gcp-ipi-proxy-private-p2-f14-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-gcp-ipi-proxy-private-p2-f14-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-p2-f14-openshift-logging-5-7-disruptive ci/rehearse/periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-sdn-workers-rhel8-p1-f7-openshift-logging-5-7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-ipi-sdn-p3-f28-openshift-logging-5-7-disruptive periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-azure-ipi-fips-p1-f7

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.6-azure-ipi-fips-p1-f3

File: ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml [hunk 1]
```diff
@@ -10,12 +10,12 @@ ref:
   - name: EO_SUB_PACKAGE
     documentation: The package name of the Elasticsearch operator to install.
     default: elasticsearch-operator
-  - name: EO_SUB_SOURCE
+  - name: SOURCE
```
Comment (anpingli) [inline ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml:13]: How about keep the channel/source independently  in cases of we have such requirement in future.

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-azure-ipi-fips-p1-f7-openshift-logging-5-7-web

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.7-azure-ipi-fips-p1-f7-openshift-logging-5-7-web

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-cco-manual-sts-p1-f7 periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-upi-p3-f28-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.11-amd64-nightly-openshift-logging-5.7-aws-ipi-proxy-cco-manual-sts-p1-f7 periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-upi-p3-f28-disruptive periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-openshift-logging-5.7-gcp-ipi-proxy-etcd-encryption-p2-f7

File: ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml (hunk 1 above)
Comment (QiaolingTang) [reply ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml:13]: I can do that, but we need to update the code in logging web console testing too.

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.6-azure-ipi-fips-p1-f3 periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-upi-p3-f28-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-openshift-logging-5.6-azure-ipi-fips-p1-f3 periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-upi-p3-f28-disruptive

File: ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml (hunk 1 above)
Comment (QiaolingTang) [reply ci-operator/step-registry/operatorhub/subscribe/elasticsearch-operator/operatorhub-subscribe-elasticsearch-operator-ref.yaml]: Done.

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-arm64-nightly-openshift-logging-5.7-azure-upi-p3-f28-disruptive

**Comment by QiaolingTang** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-multi-nightly-openshift-logging-5.7-aws-ipi-ovn-ipsec-p1-f7

**Pull request merged**